### DISCUSSION TITLE
Electrode height and width

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3697] [Q&A]
I know the plate area, how can I calculate the electrode height and width? My plate area is 1.84 m^2 for a 12 Ah NCA pouch cell.

**Comment by brosaplanella** [comment]
Unless you are using a pouch thermal model, what matters is the plate area (i.e. height * width) so you can make define them to anything that meets this requirement: e.g. height = 1.84 m and width = 1 m.
